MEASUREMENT_INCORPORATION.s

```diff
@@ -362,7 +362,7 @@ FAZC		CALL
 			GRP2PC
 # Page 1259
 		VLOAD	VAD		# START 3RD PHASE OF INCORP2
-			X789		# 7TH, 8TH, 9TH COMPONENTN OF STATE VECTOR
+			X789		# 7TH, 8TH, 9TH COMPONENTS OF STATE VECTOR
```

Comment:
Multiple issues here.

1. `COMPONENT` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1259.jpg)
2. There are no spaces between `7th` and `COMPONENT` - i.e `7TH,8TH,9TH,COMPONENT`